lib/src/controls/navigation/tree_view.dart

```diff
@@ -382,6 +383,9 @@ class TreeView extends StatefulWidget {
   /// Called when an item is invoked
   final Future<void> Function(TreeViewItem item)? onItemInvoked;
 
+  /// right click
```

Comment:
Improve this documentation

```suggestion
  /// A tap with a secondary button has occurred.
```